cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,34 @@
+ #!/usr/bin/env bash
+
+set -xe
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function pod_preset::enable_feature() {
+    local -r cluster_name=$1
+
+    docker exec "$cluster_name-control-plane" bash -c 'sed -i \
+    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
+    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
+    /etc/kubernetes/manifests/kube-apiserver.yaml'
+}
+
+function pod_preset::create_virt_launcher_fake_product_uuid_podpreset() {
+    local -r namespace=$1
+
+    if ! _kubectl get ns "$namespace" &>2; then
+        _kubectl create ns "$namespace"
+    fi
+
+    _kubectl apply -f "$KIND_MANIFESTS_DIR/product-uuid-podpreset.yaml" -n "$namespace"
+}
+
+function pod_preset::expose_unique_product_uuid_per_node() {
+    local -r cluster_name=$1
+    local -r namespace=$2
+
+    pod_preset::enable_feature "$cluster_name"
+    sleep 3
```

Comment:
is there something else beside sleep that can check the feature is enabled (best option) ?
maybe even retry on create_virt_launcher_fake_product_uuid_podpreset would be better?

Reply:
k8s does not provide proper way to check which features are enabled other then do some "forensics" to kube-apiserver
I will add a check for kube-apiserver process arguments (ps -ef | grep ..)

Reply:
Done